src/MagnumPlugins/GltfImporter/Test/GltfImporterTest.cpp

```diff
@@ -4492,6 +4495,53 @@ void GltfImporterTest::meshUnorderedAttributes() {
     CORRADE_COMPARE(mesh->attributeFormat(customAttribute1), VertexFormat::Vector3);
 }
 
+void GltfImporterTest::meshMorphTargetAttributes() {
+    Containers::Pointer<AbstractImporter> importer = _manager.instantiate("GltfImporter");
+
+    CORRADE_VERIFY(importer->openFile(Utility::Path::join(GLTFIMPORTER_TEST_DIR, "mesh-morph-target-attributes.gltf")));
+    CORRADE_COMPARE(importer->meshCount(), 1);
+
+    Containers::Optional<Trade::MeshData> mesh = importer->mesh(0);
+    CORRADE_VERIFY(mesh);
+    CORRADE_COMPARE(mesh->attributeCount(), 6);
+
+    /* Base mesh (position, normal and color) */
+    CORRADE_COMPARE(mesh->attributeCount(-1), 2);
+
+    CORRADE_VERIFY(mesh->hasAttribute(MeshAttribute::Position));
+    CORRADE_COMPARE(mesh->attributeCount(MeshAttribute::Position), 1);
+    CORRADE_COMPARE(mesh->attributeFormat(MeshAttribute::Position), VertexFormat::Vector3);
+
+    CORRADE_VERIFY(mesh->hasAttribute(MeshAttribute::Normal));
+    CORRADE_COMPARE(mesh->attributeCount(MeshAttribute::Normal), 1);
+    CORRADE_COMPARE(mesh->attributeFormat(MeshAttribute::Normal), VertexFormat::Vector3);
+
+    /* First morph target (position and normal) */
+    CORRADE_COMPARE(mesh->attributeCount(0), 2);
+
+    CORRADE_VERIFY(mesh->hasAttribute(MeshAttribute::Position, 0));
+    CORRADE_COMPARE(mesh->attributeCount(MeshAttribute::Position, 0), 1);
+    CORRADE_COMPARE(mesh->attributeFormat(MeshAttribute::Position, 0, 0), VertexFormat::Vector3);
```

Comment:
To ensure the accessor IDs aren't accidentally mixed up, can you check also all attribute content? I.e.,

```cpp
    CORRADE_COMPARE_AS(mesh->attribute<Vector3>(MeshAttribute::Position, 0), Containers::arrayView<Vector4>({
        {0.1f, 0.2f, 0.3f},
        {0.4f, 0.5f, 0.6f},
        {0.7f, 0.8f, 0.9f}
    }), TestSuite::Compare::Container);
```

and similarly for others, including the non-morph attributes.

Ideally, to differentiate further, they would also have different contents and types, let's say `Vector3sNormalized` + `Vector3bNormalized` for the extra normals and `Vector3us` for extra positions. See the `mesh-primitives-types.gltf` and `mesh-primitives-types.bin.in` files for an example.